sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -550,6 +557,7 @@ export interface NotificationHubsResponse {
 // @public
 export class NotificationHubsServiceClient {
     constructor(connectionString: string, hubName: string, options?: NotificationHubsClientOptions);
+    beginSubmitNotificationHubJob(notificationHubJob: NotificationHubJob, options?: OperationOptions): Promise<SimplePollerLike<OperationState<NotificationHubJob>, NotificationHubJob>>;
```

Comment:
nitpick: If you prefer, and you feel like the inline type is too long, you could alias it to something like `<method name>Poller`

Reply:
Added NotificationHubJobPoller type